cmd/contractcmd/init_validator_manager.go

```diff
@@ -132,28 +168,49 @@ func initValidatorManager(_ *cobra.Command, args []string) error {
 			return err
 		}
 	}
-	sc, err := app.LoadSidecar(chainSpec.BlockchainName)
-	if err != nil {
-		return fmt.Errorf("failed to load sidecar: %w", err)
-	}
-	if sc.Networks[network.Name()].ValidatorManagerAddress == "" {
-		return fmt.Errorf("unable to find Validator Manager address")
+
+	if specializedValidatorManagerAddressStr == "" && sc.UseACP99 && sc.PoS() {
```

Comment:
seems that since even in deploy validator manager we are deploying v2 validator manager, we should stop supporting non v2 contracts